src/dbcp/transform/helpers.py

```diff
@@ -241,66 +257,66 @@ def parse_dates(series: pd.Series, expected_mean_year=2020) -> pd.Series:
         return multiformat_string_date_parser(series)
 
 
-def _geocode_row(
-    ser: pd.Series, client: GoogleGeocoder, state_col="state", locality_col="county"
-) -> List[str]:
-    """Function to pass into pandas df.apply() to geocode state/locality pairs.
+def _geocode_and_add_fips(
```

Comment:
I moved some of the geocoding logic around so I could compare the Google and Geocodio results.